blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md

```diff
@@ -0,0 +1,300 @@
+
+**Execute corporate acts:** Allows the issuer to input some details and create a corporate action, for which the mechanism determines rights, schedules communications, distributes capital (if necessary), and updates records. Tokenization significantly reduces the costs of these corporate actions.
+  
+
+## The life cycle of an asset
+
+The life cycle of an asset that has been tokenized on the blockchain typically includes the following stages:
+
+**Creation:** The asset is tokenized by creating a smart contract on the blockchain that represents the asset and defines its ownership and transfer rules. A unique token is issued for the asset and stored in the smart contract.
+
+**Verification:** The ownership and authenticity of the asset are verified through the use of documents, certificates, or other forms of evidence.
+
+**Trading:** The asset is listed for sale on a marketplace or exchange that supports the trading of tokenized assets. Potential buyers can purchase the asset using the token that represents it.
+
+**Transfer:** When the asset is sold, the token representing the asset is transferred from the seller's wallet to the buyer's wallet. This transfer is recorded on the blockchain and is secured by the smart contract that was created during the creation stage.
+
+**Settlement:** After the transfer is complete, the asset is considered to be in the possession of the buyer. The settlement process may involve the transfer of physical ownership of the asset, or it may simply involve the transfer of ownership rights as represented by the token.
+
+**Redeemed:** In some cases, an asset may be redeemed by the original owner or by a third party. This process typically involves the transfer of the token back to the original owner or to the third party, and may be accompanied by the transfer of physical ownership or ownership rights.
+
+It is important to note that the specific details of the life cycle of a tokenized asset will depend on the type of asset being tokenized and the terms of the smart contract that represents it.
+
+## Tokenize a physical asset on Celo
+
+To tokenize a physical asset on the Celo blockchain, you will need to follow these steps:
+
+*Determine the type of asset* you want to tokenize and ensure that it is allowed on the Celo platform. Some examples of assets that can be tokenized on Celo include real estate, precious metals, and works of art.
+
+*Choose the appropriate Ethereum Request for Comment (ERC) standard* for representing your asset on the blockchain. There are several different ERC standards that can be used for tokenizing assets, including ERC-721 and ERC-20.
```

Comment:
*Choose the appropriate Ethereum Request for Comment (ERC) standard* for representing your asset on the blockchain. Several different ERC standards can be used for tokenizing assets, including ERC-721 and ERC-20.

Reply:
Done